lib/facility_factory.rb

```diff
@@ -0,0 +1,58 @@
+class FacilityFactory
```

Comment:
I like how you've handled formatting the data in this class. The alternative I sometimes see is a method per data source, so something like `def format_oregon_data`, `def format_ny_data`, etc. either works just fine. I could see the way it is set up here getting tedious if you have to handle all 50 states. At some point you'd probably want to switch to formatting per state rather than adding another `elsif` per state.